src/highrl/envs/eval_env.py

```diff
@@ -3,98 +3,181 @@
-        self.maximum_distance = config.getfloat("reward", "maximum_distance")
-        self.velocity_std = config.getfloat("reward", "velocity_std")
-        self.alpha = config.getfloat("reward", "alpha")
-        self.progress_discount = config.getfloat("reward", "progress_discount")
-
-        self.render_each = config.getint("render", "render_each")
-        self.save_to_file = config.getboolean("render", "save_to_file")
-
-        self.epsilon = config.getint("env", "epsilon")
-        self.collect_statistics = config.getboolean("statistics", "collect_statistics")
-        self.scenario = config.get("statistics", "scenario")
-
-    def obstacles_configure(self) -> None:
-        self._generate_obstacles_points(
-            self.num_of_hard_obstacles,
-            min_dim=200,
-            max_dim=400,
-        )
-        self._generate_obstacles_points(
-            self.num_of_medium_obstacles,
-            min_dim=150,
-            max_dim=300,
+        self.cfg.n_eval_episodes = config.getint("eval", "n_eval_episodes")
+        self.add_boarder_obstacles()
+        robot_init_x_pos = config.getint("eval", "robot_init_x_pos")
+        robot_init_y_pos = config.getint("eval", "robot_init_y_pos")
+        goal_x_pos = config.getint("eval", "goal_x_pos")
+        goal_y_pos = config.getint("eval", "goal_y_pos")
+        self.set_robot_position(
+            Position[int](robot_init_x_pos, robot_init_y_pos),
+            Position[int](goal_x_pos, goal_y_pos),
         )
-        self._generate_obstacles_points(
-            self.num_of_small_obstacles,
-            min_dim=50,
-            max_dim=150,
+        big_obs_count = config.getint("eval", "big_obs_count")
+        med_obs_count = config.getint("eval", "med_obs_count")
+        sml_obs_count = config.getint("eval", "sml_obs_count")
+
+        # big_obs_pos = config.get("eval", "big_obs_pos")
+        # med_obs_pos = config.get("eval", "med_obs_pos")
+        # sml_obs_pos = config.get("eval", "sml_obs_pos")
+
+        big_obs_pos = [(180, 180), (125, 125)]
+        med_obs_pos = [(125, 70), (70, 125), (200, 50), (150, 30)]
+        sml_obs_pos = [(30, 10), (10, 50), (100, 80), (20, 170)]
+
+        big_obs_dim = config.getint("eval", "big_obs_dim")
+        med_obs_dim = config.getint("eval", "med_obs_dim")
+        sml_obs_dim = config.getint("eval", "sml_obs_dim")
+
+        obstacles = self.generate_eval_obstacles(
+            big_obs_count,
+            med_obs_count,
+            sml_obs_count,
+            big_obs_pos,
+            med_obs_pos,
+            sml_obs_pos,
+            big_obs_dim,
+            med_obs_dim,
+            sml_obs_dim,
         )
 
-    def _generate_obstacles_points(
-        self, obstacles_count: int, min_dim: int, max_dim: int
-    ) -> None:
-        """Generate obstacles based on teacher action for next robot session
+        for obstacle in obstacles:
+            self.obstacles.obstacles_list.append(obstacle)
```

Comment:
Maybe this logic should be added as a function in the `highrl.obstacles.Obstacles` class? 
```python
Class Obstacles:
...
   def add_obstacles(self, obstacles_list: List[SingleObstacle]) -> None:
       """Add new obstacles to the current obstacles list"""
       self.obstacles_list.extend(obstacles_list)
```
This way the implementation detail (i.e., the obstacles are store in a list) is hidden from the user.